fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -492,6 +493,36 @@ Future<Void> configureTenantTransaction(Transaction tr,
 	return Void();
 }
 
+ACTOR template <class Transaction>
+Future<Void> changeLockState(Transaction* tr, int64_t tenant, TenantLockState desiredLockState, UID lockID) {
+	state Optional<TenantMapEntry> entry;
+	tr->setOption(FDBTransactionOptions::RAW_ACCESS);
+	tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+	tr->setOption(FDBTransactionOptions::LOCK_AWARE);
+	tr->setOption(FDBTransactionOptions::READ_LOCK_AWARE);
+	wait(store(entry, TenantAPI::tryGetTenantTransaction(tr, tenant)));
+	ASSERT(entry.present());
```

Comment:
There is also a function `TenantAPI::getTenantTransaction` which will throw a `tenant_not_found` error if the tenant isn't present. If you want this to be an assert, then what you have is probably preferable, but I just wanted to throw that out there.